frontend/src/BenchmarkTest.res

```diff
@@ -68,6 +80,11 @@ let renderMetricOverviewRow = (
         )
       | _ => ("NA", "NA")
       }
+      let color = switch row.delta {
+      | Some(delta) =>
+        (delta >= 0. && row.ascending) || (delta <= 0. && !row.ascending) ? Sx.green300 : Sx.red300
```

Comment:
The equality `( = )` operator can be used on boolean to mean "both or neither" : `(delta >= 0.) = row.ascending` (... it's a fun trick, but it probably makes the code harder to read if you don't know it, so feel free to leave the explicit condition!)

Reply:
This is neat trick, that's only slightly harder to read. I'll switch over.